### ISSUE TITLE
`published_parsed` field in metadata is not TZ-aware

### ISSUE THREAD

**Issue by jdangerx** [catalyst-cooperative/pudl-archiver#193]
The `published_parsed` value used in `src/pudl_archiver/archivers/ferc/xbrl.py` to describe the publishing time of an XBRL filing is currently not timezone aware. While our main use case for this field is relative ordering, which doesn't require the timezone to be correct, we generate a `published_time` field in our XBRL extraction outputs down the line, which one would expect to be the *correct* time.

```[tasklist]
## Scope
* [ ] `published_parsed` includes timezone from XBRL rss feed
```

**Comment by mariahjrogers** [comment]
👋🏻 Hello! I was interested in picking up a "good first issue" ticket and saw this, but I also see it may have been handled in [this commit ](https://github.com/catalyst-cooperative/pudl-archiver/pull/362/commits/81adb6e5167935b5eec816f50278c234e3580fe1) of #362 ?

Can you confirm whether this is still needed?

**Comment by jdangerx** [comment]
Yes, good sleuthing! This was handled by #362 and I forgot to close the issue. Let me see if I can find something else that might be a good intro to contributing - are you primarily interested in the archiver?
